### PR TITLE
Login Issues

### LINKED ISSUE #366: Email confirmation page should redirect to login

### PR DESCRIPTION
**Pull request by Ommy**
Now when a user signs up, it redirects them to the logged in page. For email confirmation, added a view that hits the backend to confirm user's email. It then redirects to the appropriate page depending on session.

Resolves #366 
Squashes #373

### CODE REVIEW

File: src-backbone/app/js/views/auth/confirmEmailView.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                App().RootView.clearNotifications();
+                App().RootView.showNotification({
+                    title: 'There was a problem',
+                    desc: "Your email couldn't be verified"
+                });
+            }
+        });
+    },
+
+    navigateToAppropriateScreen: function() {
```
Comment (Trinovantes) [inline src-backbone/app/js/views/auth/confirmEmailView.js]: rename this to navigateToDefaultScreen

File: src-backbone/app/js/models/session.js [hunk 2]
```diff
@@ -70,6 +70,12 @@ let SessionModel = Backbone.Model.extend({
                 App().RootView.hideSpinner();
             },
             success: function(response) {
+                App().RootView.clearNotifications();
+                App().RootView.showNotification({
+                    title: 'Success!',
+                    desc: 'An email has been sent to your provided email address. Please follow the link to verify your email.',
```
Comment (Kapin) [inline src-backbone/app/js/models/session.js]: probably should i18n this

File: src-backbone/app/js/views/auth/confirmEmailView.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            },
+            complete: function() {
+                App().RootView.hideSpinner();
+            },
+            success: function(response) {
+                self.navigateToDefaultScreen();
+                App().RootView.clearNotifications();
+                App().RootView.showNotification({
+                    title: 'Success!',
+                    desc: 'Email confirmed!',
```
Comment (Kapin) [inline src-backbone/app/js/views/auth/confirmEmailView.js]: i18n

File: src-backbone/app/js/views/auth/confirmEmailView.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                    alertType: 'success',
+                });
+            },
+            error: function(errors) {
+                self.navigateToDefaultScreen();
+                App().RootView.clearNotifications();
+                App().RootView.showNotification({
+                    title: 'There was a problem',
+                    desc: "Your email couldn't be verified"
```
Comment (Kapin) [inline src-backbone/app/js/views/auth/confirmEmailView.js]: i18n if possible

**Comment by Kapin** [comment]
just some questions around translations, but otherwise lgtm

File: src-backbone/app/js/views/auth/confirmEmailView.js (hunk 4 above)
Comment (Ommy) [inline src-backbone/app/js/views/auth/confirmEmailView.js]: k

File: src-backbone/app/js/views/auth/confirmEmailView.js (hunk 3 above)
Comment (Ommy) [inline src-backbone/app/js/views/auth/confirmEmailView.js]: done

File: src-backbone/app/js/models/session.js (hunk 2 above)
Comment (Ommy) [inline src-backbone/app/js/models/session.js]: done

File: src-backbone/app/js/views/auth/confirmEmailView.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+                App().RootView.showSpinner();
+            },
+            complete: function() {
+                App().RootView.hideSpinner();
+            },
+            success: function(response) {
+                self.navigateToDefaultScreen();
+                App().RootView.clearNotifications();
+                App().RootView.showNotification({
+                    title: 'Success!',
```
Comment (Trinovantes) [inline src-backbone/app/js/views/auth/confirmEmailView.js]: why aren't the titles i18n?

**Pull request merged**